.github/workflows/rabbit_consumer.yaml

```diff
@@ -1,13 +1,13 @@
-name: Test and Lint
+name: Rabbit Consumer
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
-  build:
+  test_and_lint:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.10"]
```

Comment:
Why have we removed python 3.8 and 3.9 from the list of python versions here?

Reply:
We only build the docker file against 3.10, so it's wasted checking these older versions which we won't run against

Good question :+1: